### PR TITLE
feat(sentry): implement sentry

### PR DESCRIPTION
**Pull request by sirjla**
# Resumen
Se implemento sentry

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -9,3 +10,8 @@ Vue.config.productionTip = false;
 new Vue({
   render: h => h(App),
 }).$mount('#app');
+
+Sentry.init({
+  dsn: 'https://279544681984435792b43230421d4610@sentry.io/1369885',
+  integrations: [new Sentry.Integrations.Vue({ Vue })]
+})
```
Comment (monkeyci) [inline src/main.js]: Missing semicolon                                     semi

File: src/main.js [hunk 2]
```diff
@@ -9,3 +10,8 @@ Vue.config.productionTip = false;
 new Vue({
   render: h => h(App),
 }).$mount('#app');
+
+Sentry.init({
+  dsn: 'https://279544681984435792b43230421d4610@sentry.io/1369885',
+  integrations: [new Sentry.Integrations.Vue({ Vue })]
```
Comment (monkeyci) [inline src/main.js]: Missing trailing comma                                comma-dangle

File: src/main.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import Vue from 'vue';
 import dotenv from 'dotenv';
 import App from './App.vue';
+import * as Sentry from '@sentry/browser'
```
Comment (monkeyci) [inline src/main.js]: Absolute imports should come before relative imports  import/imports-first<br>Missing semicolon                                     semi

**Comment by sirjla** [comment]
[Integrar Sentry](https://trello.com/c/7gDWZO4h/95-integrar-sentry)

File: src/main.js [hunk 4]
```diff
@@ -9,3 +10,8 @@ Vue.config.productionTip = false;
 new Vue({
   render: h => h(App),
 }).$mount('#app');
+
+Sentry.init({
+  dsn: 'https://279544681984435792b43230421d4610@sentry.io/1369885',
```
[the remark this answers is not in the document]
Comment (sirjla) [reply src/main.js]: es de mi cuenta de sentry por ahora

File: src/main.js [hunk 5]
```diff
@@ -1,11 +1,21 @@
+import * as Sentry from '@sentry/browser'
```
Comment (monkeyci) [inline src/main.js]: Missing semicolon  semi

**Pull request merged**